src/mountscript.sh

```diff
@@ -217,13 +225,47 @@ get_dir_from_share()
 
     if [ -z "$account" -o -z "$container" -o -n "$extra" ]; then
         eecho "Bad share name: ${hostshare}."
-        eecho "Share to be mounted must be of the form 'account.blob.core.windows.net:/account/container'."
+        eecho "Share to be mounted must be of the form 'account.$2.core.windows.net:/account/container'."
         return 1
     fi
 
     echo "$share"
 }
 
+get_version_from_MOUNT_OPTIONS()
+{
+    local mount_options="$MOUNT_OPTIONS"
+    local ver_string="vers="
+    local minor_ver_string="minorversion"
+    local nfs_ver=""
+    local nfs_minorvers=""
+
+    if [ -z "$mount_options" ] || [[ ! "$mount_options" == *"$ver_string"* ]]; then
```

Comment:
Please confirm the previous v3 changes, and make sure no option means using v3. Once confirmed, add a comment for this if check.

Reply:
Discussed with Raymond offline. v3 adds vers=3 option in mount command if no version is given. Hence defaulting vers=3 here to avoid breaking changes for v3.